.isort.cfg

```diff
@@ -23,3 +23,4 @@ skip=
     ,hydra/grammar/gen
     ,tools/configen/example/gen
     ,tools/configen/tests/test_modules/expected
+    ,temp
```

Comment:
`temp` looks like it's not supposed to be here :)

Reply:
Oh, but it does. :P
I am using temp as stating area for bug repros and other crap and I don't need it linted.